doozer/doozerlib/backend/konflux_olm_bundler.py

```
@@ -366,139 +465,211 @@ def _get_digest_image_pattern():
+            image_info_coros.append(
+                util.oc_image_info_for_arch_async(
+                    build_pullspec,
+                    registry_config=os.getenv("QUAY_AUTH_FILE"),
+                )
+            )
+
+        # Phase 2: Fetch all image infos concurrently
+        image_infos = await asyncio.gather(*image_info_coros)
+
+        resolved: dict[str, tuple[str, str, str]] = {}
+        for (name, spec, _build_pullspec), image_info in zip(operand_entries, image_infos):
```

Ruff (0.15.18)

[warning] Loop control variable `name` not used within loop body

(B007)